### PR TITLE
Update the README to reflect location of bounties

### PR DESCRIPTION
**Pull request by JST5000**
A bit more detail would be helpful to connect the dots between new viewers and finding the individual bounties.

### CODE REVIEW

**Review by intelliot — APPROVED** [review]

**Review by ckniffen — APPROVED** [review]

**Comment by JST5000** [comment]
Once this gets 3 approvers, please merge it - I won't be checking email until tomorrow :)

**Review by pdp2121 — APPROVED** [review]

**Pull request merged**
